### PR TITLE
[JBPM-10106] Orphaned records in the sessionInfo table

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-10106

### CODE REVIEW

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
Jenkins run fdb

**Comment by elguardian** [comment]
Jenkins run fdb

**Comment by elguardian** [comment]
Jenkins run fdb

File: jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManagerTest.java [hunk 1]
```diff
@@ -1568,7 +1568,7 @@ public void testExceptionWhenCompleteTaskAfterEngineDisposal() throws Exception
         emf.close();
     }
 
-    @Test(timeout=15000)
+    @Test //(timeout=15000)
```
Comment (afalhambra) [inline jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManagerTest.java:1571]:
```suggestion
    @Test
```

**Review by afalhambra — COMMENTED** [review]
would be nice to have a test where we can check this functionality

File: jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManagerTest.java (hunk 1 above)
Comment (martinweiler) [reply jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManagerTest.java:1571]: I have created a test case here:
https://github.com/kiegroup/jbpm/commit/837f19b7b0ead384c1c982f1e7e733eb4e3bac39

For the use case of a signal being sent without a processInstanceId, the additional SessionInfo entry is created as a consequence of this code:
https://github.com/kiegroup/jbpm/blob/main/jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManager.java#L164-L172

@elguardian the code change from this PR does not seem to address this.

**Pull request closed without merging**